### PR TITLE
Moving symlinks spec from SPECS-EXTENDED to SPECS directory.

### PR DESCRIPTION
**Pull request by Sumynwa**
Moving symlinks spec from SPECS-EXTENDED to SPECS directory.

###### Merge Checklist  

**All** boxes should be checked before merging the PR *(just tick any boxes which don't apply to this PR)*
- [x] The toolchain has been rebuilt successfully (or no changes were made to it)
- [x] The toolchain/worker package manifests are up-to-date
- [x] Any updated packages successfully build (or no packages were changed)
- [x] Packages depending on static components modified in this PR (Golang, `*-static` subpackages, etc.) have had their `Release` tag incremented.
- [x] Package tests (%check section) have been verified with RUN_CHECK=y for existing SPEC files, or added to new SPEC files
- [x] All package sources are available
- [x] cgmanifest files are up-to-date and sorted (`./cgmanifest.json`, `./toolkit/tools/cgmanifest.json`, `./toolkit/scripts/toolchain/cgmanifest.json`, `.github/workflows/cgmanifest.json`)
- [x] LICENSE-MAP files are up-to-date (`./SPECS/LICENSES-AND-NOTICES/data/licenses.json`, `./SPECS/LICENSES-AND-NOTICES/LICENSES-MAP.md`, `./SPECS/LICENSES-AND-NOTICES/LICENSE-EXCEPTIONS.PHOTON`)
- [x] All source files have up-to-date hashes in the `*.signatures.json` files
- [x] `sudo make go-tidy-all` and `sudo make go-test-coverage` pass
- [x] Documentation has been updated to match any changes to the build system
- [x] Ready to merge

---

###### Summary 
Moving symlinks to SPECS from SPECS-EXTENDED.
Adding as run dependency for package cassandra medusa.

###### Change Log  
- Modifying spec file based on Fedora 36.
- Not updating LICENSES-MAP.md/licenses.json or cgmanifest as already updated.

###### Does this affect the toolchain?  
NO

###### Associated issues  

- #xxxx

###### Links to CVEs  
- https://nvd.nist.gov/vuln/detail/CVE-YYYY-XXXX

###### Test Methodology

- Local Build

### CODE REVIEW

File: SPECS/symlinks/symlinks.spec [hunk 1]
```diff
@@ -4,10 +4,10 @@ Vendor:         Microsoft Corporation
 Distribution:   Mariner
 URL: http://ibiblio.org/pub/Linux/utils/file/
 Version: 1.7
-Release: 3%{?dist}
+Release: 7%{?dist}
 License: Copyright only
 # Upstream maintainer provided tarball, ibiblio no longer allowing uploads
-Source0: http://ibiblio.org/pub/Linux/utils/file/%{name}-%{version}.tar.gz
+Source0: http://ibiblio.org/pub/Linux/utils/file/%{name}-%{version}.tar.gz#/%{name}-%{version}.tar.gz
```
Comment (oliviacrain) [inline SPECS/symlinks/symlinks.spec:10]: This URL seems to be dead. If that's the case, then we should do the following:
- Move the original URL to a comment
- Change Source0 to point to `%{_mariner_sources_url}/%{name}-%{version}.tar.gz`
- Update the `cgmanifest.json` file to point to our sources blobstore

**Review by oliviacrain — CHANGES_REQUESTED** [review]
In addition to the source URL issue, please apply the linter's suggestions to this spec file.

File: SPECS/symlinks/symlinks.spec (hunk 1 above)
Comment (Sumynwa) [reply SPECS/symlinks/symlinks.spec]: Done.

File: SPECS/symlinks/symlinks.spec [hunk 2]
```diff
@@ -41,6 +42,22 @@ install -m 644 symlinks.1 $RPM_BUILD_ROOT%{_mandir}/man1
 %{_mandir}/man1/symlinks.1*
 
 %changelog
```
Comment (oliviacrain) [inline SPECS/symlinks/symlinks.spec:44]: Please lint the spec to be consistent with our other specs.

**Review by oliviacrain — APPROVED** [review]
Approving- feel empowered to merge when ready, but I would appreciate some linting of the spec.

File: SPECS/symlinks/symlinks.spec (hunk 2 above)
Comment (Sumynwa) [reply SPECS/symlinks/symlinks.spec:43]: Done

**Pull request merged**